metpy/calc/thermo.py

```diff
@@ -222,7 +222,7 @@ def lfc(pressure, temperature, dewpt):
     parcel_profile
     """
     ideal_profile = parcel_profile(pressure, temperature[0], dewpt[0]).to('degC')
-    x, y = find_intersections(pressure, ideal_profile, temperature)
+    x, y = find_intersections(pressure[1:], ideal_profile[1:], temperature[1:])
```

Comment:
Can you add a comment on why you're skipping the first point (and why it's ok)? I feel like otherwise we'll keep passing by this line and have to stop and wonder why.